sdk/keyvault/azsecrets/internal/keyvault_client.go

```diff
@@ -33,7 +34,7 @@ func NewKeyVaultClient(options *azcore.ClientOptions) *KeyVaultClient {
 		options = &azcore.ClientOptions{}
 	}
 	client := &KeyVaultClient{
-		pl: runtime.NewPipeline(moduleName, moduleVersion, runtime.PipelineOptions{}, options),
+		Pl: runtime.NewPipeline(ModuleName, ModuleVersion, runtime.PipelineOptions{}, options),
```

Comment:
This gives requests the wrong User-Agent: `azsdk-go-internal/v0.7.0 ...`

Reply:
fixed to `azsdk-go-azsecrets/v0.7.0` is that correct?

Reply:
I believe so. @kimprice @mikekistler is `azsdk-go-azsecrets/v0.7.0 azsdk-go-azcore/v0.23.0 (go1.18; Windows_NT)` the right format for our telemetry?

Reply:
That looks right according to this: https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy